data/json/overmap_specials.json

```diff
@@ -534,6 +534,26 @@
         "flags" : [ "CLASSIC" ]
     },{
         "type" : "overmap_special",
+        "id" : "Office Tower 2.0",
+        "overmaps" :
+        [
+            { "point":[-2,0,0], "overmap": "office_tower_2_a1_north"},
+            { "point":[-1,0,0], "overmap": "office_tower_2_a2_north"},
+            { "point":[0,0,0], "overmap": "office_tower_2_a3_north", "connect": "road"},
+            { "point":[-2,1,0], "overmap": "office_tower_2_b1_north"},
+            { "point":[-1,1,0], "overmap": "office_tower_2_b2_north"},
+            { "point":[0,1,0], "overmap": "office_tower_2_b3_north"}
+        ],
+        "locations" : [ "wilderness" ], "//":"what special locations does it spawn",
+        "city_distance" : [-1, 4], "//":"how far from a city it should be",
+        "city_sizes" : [6, 12], "//":"what city sizes should it spawn in",
+        "occurrences" : [1, 5], "//":"how many per overmap",
```

Comment:
oops, this causes the special-crowding--min 1 is similar to required = true.  0-5 would be better.